tests/c/internal_linkage_same_obj.c

```diff
@@ -16,7 +24,7 @@ static int call_me(int x) {
 }
 
 int main(int argc, char **argv) {
-  YkMT *mt = yk_mt_new(NULL);
+  YkMT *mt = yk_mt_new();
```

Comment:
This is wrong: `yk_mt_new` should take `NULL` here.

Reply:
This would have failed a compile-time check, so it didn't work.

Reply:
It never ran cause it had the ignore tag. My bad :facepalm: